wazuh/wazuh-dashboard-plugins
# Fix reports path in toast

### Description
After the redesign of the application, the path of generated reports changed. This PR changes the toast that notifies the user where to find the reports to show the current path.
 
### Issues Resolved
#6215

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/42900763/54801186-648b-4f86-aaa4-09620d53b0e7)


### Test
 -  Navigate to IT Hygiene 
 - Select an agent
 - Go to Configuration option
 - Select Generate Report
 - Confirm that the toast shows the correct path

### Check List
